### PR TITLE
Wip lesson

### LINKED ISSUE #1: new content in progress
[Outline here](https://github.com/learn-co-curriculum/sinatra-fwitter-2-forms/blob/master/README.md)

### PR DESCRIPTION
**Pull request by ipc103**
closes #1 - ready for review @AnnJohn @deniznida @dfenjves @SophieDeBenedetto

### CODE REVIEW

**Comment by AnnJohn** [comment]
@deniznida will review

Deadline- today, 9/11/15

File: README.md [hunk 1]
```diff
@@ -1,24 +1,136 @@
...
+  <body> 
+      <h1>Add a tweet</h1>
+      <form action="/tweet" method="POST">
+
+      </form>
+
+  </body>
+</html>
+```
+Our form needs two inputs - one for `username` and one for status. We'll use a type of `text` for each one, as well as `name`. The `name` attribute is very important - this will be the key in the `params` hash when the user makes their post request. We can make the `name` attribute whatever we want, but `username` and `status` make sense. Finally, we also need a Submit button so users can submit the form.
```
Comment (deniznida) [inline README.md]: back-ticks around status are missing.

Our form needs two inputs - one for `username` and one for `status`.

**Comment by deniznida** [comment]
@ipc103 added back-ticks and merging

**Comment by deniznida** [comment]
closes issue #1

**Pull request merged**